src/main/java/com/safehiring/ems/controller/RegistrationController.java

```diff
@@ -141,7 +141,7 @@ public JwtResponse generateJwtToken(@RequestBody final JwtRequest jwtRequest) {
              * DisabledException not working. That means If a user is inactive, Still we are not geeting this
              * DisabledException error
              */
-            throw new InvalidUserException("User Inactive");
+            throw new InvalidUserException("Please verify your email by clicking on the link on the email");
```

Comment:
"User Inactive, Please verify your email by clicking on the link sent on the email to proceed. If facing any issue, Please contact at ceo@youremployeecheck.com or whatsapp at +91-8285867446"

Reply:
Everywhere, where a string is being displayed at UI in cases like exception. Please add this line. "If facing any issue, Please contact at ceo@youremployeecheck.com or whatsapp at +91-8285867446""